charts/temporal/Chart.yaml

```diff
@@ -28,6 +28,10 @@ dependencies:
     repository: https://grafana.github.io/helm-charts
     version: 8.0.2
     condition: grafana.enabled
+  - name: mysql
```

Comment:
I think this is out-of-scope for the feature request. The only change that should be made here is updating the cassandra dependency to Bitnami from incubator. 

I see a lot of additions here for MySQL that I don't think are required. 

I would recommend scaling back the changes to only that which satisfies the feature request in [Milestone 1](github.com/temporalio/helm-charts/issues/468)